scripts/utils.js

```diff
@@ -38,7 +38,7 @@ async function getNonce(address) {
 
 function resolveUrl() {
   if (env === 'mocknet') {
-    return 'http://localhost:20080';
+    return 'http://localhost:3999';
```

Comment:
back to `3999`?

Reply:
"Unfortunately" yes. The clarinet release went back to 3999 which is the default. Perhaps we can configure it through an ENV variable so every contributor can choose what to do